src/validate_utf8.ts

```diff
@@ -18,7 +16,7 @@ const CONTINUING_CHAR = 0x80;
  * @param {Number} end The index to end validating
  * @returns {boolean} True if valid utf8
  */
-function validateUtf8(bytes, start, end) {
+export function validateUtf8(bytes, start, end) {
```

Comment:
missed typings?

Reply:
I wasn't able to get the convert script we used on the driver to work here, so this is all done by hand, I figured that script just populated most of the codebase with `any` anyway so its fine to omit here and we will type properly in the next iteration.